uiowa/uiowa
# Fix error on modules overview page.

Just noticed this locally. Similar to https://github.com/uiowa/uiowa/issues/565.

### Testing
- On main, go to modules overview page locally.
- See error.
- Check out this branch.
- ??
- Profit.
